node/lib/util/submodule_util.js

```diff
@@ -610,7 +610,9 @@ exports.resolveSubmoduleNames = function (workdir,
         const relPath = GitUtil.resolveRelativePath(workdir,
                                                     cwd,
                                                     filename);
-        const result = exports.getSubmodulesInPath(relPath, submoduleNames, true);
```

Comment:
what's going on with these changes?  They look like just formatting?

Reply:
yes, somehow they total line length exceeds 80 but not complained by travis ci...